code/datums/black_market_item.dm

```diff
@@ -240,6 +240,21 @@ for anyone but the person committing mass murder.
 /datum/black_market_item/tech
 	category = "Advanced Technology" 
 
+/datum/black_market_item/tech/emagged_nikita
+	name = "Emagged Nikita"
+	desc = "A rocket launcher that fires missiles that you can ride. Great for special effects in Live-Action movies and terrorist roleplay."
+	item = /obj/structure/closet/crate/secure/weapon/experimental/nikita
+	sps_chances = list(0, 20, 35)
+	stock_min = 1
+	stock_max = 1
```

Comment:
Just one? Are you serious? How am I, a clockwork cultist chaplain, supposed to have **FUN** with all my friendly gun cult apprentices if we have only one emagged nikita? There should be _at least_ 20.